x/permission/types/keys.go

```diff
@@ -39,6 +43,9 @@ var (
 
 	PolicySequencePrefix      = []byte{0x41}
 	GroupMemberSequencePrefix = []byte{0x42}
+
+	PolicyQueueKeyPrefix      = []byte{0x51}
```

Comment:
A big problem now is that:
when a member or policy is deleted but not expired, you should delete if from the queue as well. Please check all the related logic.

Reply:
Done